app/src/main/java/com/jerboa/ui/components/comment/mentionnode/CommentMentionNode.kt

```diff
@@ -112,6 +118,16 @@ fun CommentMentionNodeFooterLine(
 ) {
     var showMoreOptions by remember { mutableStateOf(false) }
 
+    val canMod =
```

Comment:
I'd forgotten to add comment removals to comment mention nodes.

Reply:
I am in the progress of merging all those in the existing commentNodes,